lighthouse-core/config/budget.js

```diff
@@ -46,6 +46,22 @@ class Budget {
     }
   }
 
+  /**
+   * Asserts that arr has no duplicate strings, throwing an error if it does.
+   * `arrayName` is included for nicer logging.
+   * @param {Array<string>} arr
+   * @param {string} arrayName
+   */
+  static assertNoDuplicateStrings(arr, arrayName) {
```

Comment:
`arr` feels a bit of an unnecessary abbreviation 😛

Reply:
> `arr` feels a bit of an unnecessary abbreviation 😛

very fair :)